cypress/support/config/services.js

```diff
@@ -801,16 +829,31 @@ module.exports = {
   },
   serbian: {
     font: undefined,
+    isWorldService: true,
+    variant: 'lat',
```

Comment:
This setup doesn't allow easy addition of multiple variants to the Cypress config.

We'll want cypress tests for both the `lat` and `cyr` variants for the `serbian` service. Likewise for `trad` as well as `simp` variants for `zhongwen` and `ukchina`. 

I'd suggest changing the config shape to have variant value as a key, since the fonts and pageType paths can differ for different variants. 
```
  serbian: {
    isWorldService: true,
    lat: {
      font: undefined,
      pageTypes: {
        articles: {
          path:
            Cypress.env('APP_ENV') === 'live' || Cypress.env('APP_ENV') === 'test'
              ? undefined
              : '/serbian/articles/c805k05kr73o/lat',
          smoke: true,
        },
        errorPage404: {
          path:
            Cypress.env('APP_ENV') === 'live' || Cypress.env('APP_ENV') === 'test'
              ? undefined
              : '/serbian/articles/cabcdefghijo/lat',
          smoke: true,
        },
        frontPage: { path: '/serbian/lat', smoke: true },
        liveRadio: { path: undefined, smoke: false },
      },
    },
    cyr: {
      font: undefined,
      pageTypes: {
        articles: {
          path:
            Cypress.env('APP_ENV') === 'live' || Cypress.env('APP_ENV') === 'test'
              ? undefined
              : '/serbian/articles/c805k05kr73o/cyr',
          smoke: true,
        },
        errorPage404: {
          path:
            Cypress.env('APP_ENV') === 'live' || Cypress.env('APP_ENV') === 'test'
              ? undefined
              : '/serbian/articles/cabcdefghijo/cyr',
          smoke: true,
        },
        frontPage: { path: '/serbian/cyr', smoke: true },
        liveRadio: { path: undefined, smoke: false },
      },
    }
  },
```
A consequence of this restructure would be to need the `default` key for services that don't have variants. e.g.
```
  persian: {
    isWorldService: true,
    default: {
      font: 'Nassim',
      pageTypes: {
        ...
      },
    },
  },

Reply:
Chatted with @benjaminhobbs about this, soz should have put in PR. We decided to just test 1 variant for now, and later either add tests that navigate between the 2 variants, or change the config. I think this PR would become much too big If I refactor cypress tests too. Happy top raise an issue?

Reply:
Okay, I see. Yes please do raise a separate issue so we can test multiple variants in our cypress tests.

Reply:
https://github.com/bbc/simorgh/issues/3698